Makefile

```diff
@@ -74,6 +74,7 @@ PORTAL_PATH=$(BUILDPATH)/src/portal
 CHECKENVCMD=checkenv.sh
 
 # parameters
+BUILDARCH=x86_64 # default is x86_64
```

Comment:
Adding BUILDARCH may cause a result that more and more arch specific path adding to the upstream project, which is something we want to avoid.

Can we introduce a flag specifically like `BUILD_PG96` ?

Reply:
eventually, we will have a Arch parameter and ship harbor images index, can we just start it from this PR? 

Fo